hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirRenameOp.java

```diff
@@ -638,16 +654,22 @@ private static class RenameOperation {
     private final byte[] srcChildName;
     private final boolean isSrcInSnapshot;
     private final boolean srcChildIsReference;
-    private final QuotaCounts oldSrcCounts;
+    private final QuotaCounts oldSrcCountsInSnapshot;
+    private final boolean sameStoragePolicy;
+    private final Optional<QuotaCounts> srcSubTreeCountOp;
```

Comment:
These two variable names end in `Op`, which may cause misunderstandings with EditLogOp. How about calling it srcSubTreeCounts direcly?